src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -115,7 +115,7 @@
           + "    col_name TEXT,\n"
           + "    col_type TEXT NOT NULL,\n"
           + "    is_indexed INTEGER NOT NULL, \n"
-          + "    ascending_order BOOLEAN,\n"
+          + "    ascending_order INTEGER, \n"
```

Comment:
What's the meaning of this integer field? Since it's a nullable attribute I assume that it's equivalent to a nullable Boolean? If that is the case, then maybe a name that suggested boolean-ness would be better, like `is_ascending_index` or `has_ascending_index_order`? Otherwise I'm left wondering if it just represents some arbitrary encoding of index order into integers (like `+/-1`, say, rather than `0/1`).